docs/base/node_tutorial.md

```diff
@@ -102,6 +102,20 @@ app.get('/users/:id', async (req, res) => {
 });
 ```
 
+Another option would to use `Base.fetch(query)` to search for records to return, like so:
+
+```js
+app.get('/search-by-age/:age, async (req, res) => {
+    const { age } = req.params;
+    const users = await db.fetch({'age': age});
+    if (users) {
+        res.json(user);
+    } else {
+        res.status(404).json({"message": "no users ${age} years old found"});
+    }
+});
```

Comment:
I'm not sure about the code here but you don't need to check if it's empty, just return what the db gives you.
There are two possible response:

- Empty array
- Array with items

Both are acceptable response for the front end. No need to use `404`, 404 is only used when you look an item by a key IIRC.
```suggestion
    return (await db.fetch({'age': age}).next()).value;
});
```

Reply:
Makes sense to me! Good spot!